build.sh

```diff
@@ -1,19 +1,23 @@
 #!/bin/bash
 
-# Loop through each wp-module in the plugin.
-
-for DIR in wp-modules/*/; do
-	# Go to the directory of this wp-module.
-	cd "$DIR";
-	echo "Module: $DIR"
+# Run setup.
+. ./setup.sh
 
-	# Run the "npm run build" command in its package.json file.
-	if [ -f package.json ]
+# Loop through each wp-module in the plugin.
+for DIR in $plugindir/wp-modules/*; do
+	# If this module has a package.json file.
+	if [ -f "$DIR/package.json" ];
 	then
-		npm run build
+		# Go to the directory of this wp-module.
+		cd "$DIR";
+		# Run npm install for this module.
+		npm install;
```

Comment:
It might be nice to still have an `install-module-dependencies.sh` script.

It's not possible to run `lint-js.sh` without running `build.sh` first. 

It might be hard to understand for people new to the project.

Reply:
Ah, so `npm run build` [runs on](https://github.com/studiopress/fse-studio/pull/79/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R11) `preinstall`. That makes sense.

But maybe something like `install-module-dependencies.sh` could run

Reply:
Yeah I agree. We can put that file back

Reply:
In my mind I was thinking instlaling and building could be linked. But from a linting perspective they are separate!